app/port/controller/package/DownloadPackageVersionTar.ts

```
@@ -1,136 +1,419 @@
+    assert.match(res.body.rev, /^\d+\-\w{24}$/);
+
+    pkg = await TestUtil.getFullPackage({ name: scopedName, version: '1.0.0' });
+    res = await app.httpRequest()
+      .put(`/${pkg.name}`)
+      .set('authorization', publisher.authorization)
+      .set('user-agent', publisher.ua)
+      .send(pkg);
+    assert(res.status === 201);
+    assert(res.body.ok === true);
+    assert.match(res.body.rev, /^\d+\-\w{24}$/);
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)